scaleapi/__init__.py

```diff
@@ -132,55 +112,13 @@ def tasks(self, **kwargs):
         return Tasklist(docs, response['total'], response['limit'],
                         response['offset'], response['has_more'])
 
-    def create_categorization_task(self, **kwargs):
-        validate_payload('categorization', kwargs)
-        taskdata = self._postrequest('task/categorize', payload=kwargs)
-        return Task(taskdata, self)
-
-    def create_transcription_task(self, **kwargs):
-        validate_payload('transcription', kwargs)
-        taskdata = self._postrequest('task/transcription', payload=kwargs)
-        return Task(taskdata, self)
-
-    def create_phonecall_task(self, **kwargs):
-        raise ScaleException('Phone call tasks have been deprecated and are no longer available.', 400)
-
-    def create_comparison_task(self, **kwargs):
-        validate_payload('comparison', kwargs)
-        taskdata = self._postrequest('task/comparison', payload=kwargs)
-        return Task(taskdata, self)
-
-    def create_annotation_task(self, **kwargs):
-        validate_payload('annotation', kwargs)
-        taskdata = self._postrequest('task/annotation', payload=kwargs)
-        return Task(taskdata, self)
-
-    def create_polygonannotation_task(self, **kwargs):
-        validate_payload('polygonannotation', kwargs)
-        taskdata = self._postrequest('task/polygonannotation', payload=kwargs)
-        return Task(taskdata, self)
-
-    def create_lineannotation_task(self, **kwargs):
-        validate_payload('lineannotation', kwargs)
-        taskdata = self._postrequest('task/lineannotation', payload=kwargs)
-        return Task(taskdata, self)
 
-    def create_datacollection_task(self, **kwargs):
-        validate_payload('datacollection', kwargs)
-        taskdata = self._postrequest('task/datacollection', payload=kwargs)
+def _AddTaskTypeCreator(name):
+    endpoint = 'task/' + ('categorize' if name == 'categorization' else name)
```

Comment:
oh, see comment on the node sdk, we do also have a /task/categorization endpoint